axis.go

```diff
@@ -428,18 +498,17 @@ func (LogTicks) Ticks(min, max float64) []Tick {
 	if min <= 0 {
 		panic("Values must be greater than 0 for a log scale.")
 	}
-	prec := precisionOf(max)
 	for val < max*10 {
 		for i := 1; i < 10; i++ {
 			tick := Tick{Value: val * float64(i)}
 			if i == 1 {
-				tick.Label = formatFloatTick(val*float64(i), prec)
+				tick.Label = formatFloatTick(val*float64(i), displayPrecision)
 			}
 			ticks = append(ticks, tick)
 		}
 		val *= 10
 	}
-	tick := Tick{Value: val, Label: formatFloatTick(val, prec)}
+	tick := Tick{Value: val, Label: formatFloatTick(val, displayPrecision)}
```

Comment:
... and here.

Reply:
And, just a thought, I have changed the way of identification precision of LogScale, but here we can make the level of precision constant, so that really large numbers do not look ugly and overload the plot with zeros. My opinion is that scientific notation on all the Y-axis will look more elegant. And as the ticks here cannot be equal to each other (due to the way we make them), even the precision level of 1 should work, would you agree?

Reply:
That's an aesthetic decision. I think that needs discussion. Please make an issue to deal with that after this is in.